src/core/Layout/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { Component, Fragment } from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+
+import MainDrawer from '../MainDrawer';
+import RouteSwitch from '../RouteSwitch';
+import './styles.scss';
+
+class Layout extends Component {
+  render() {
+    return (
+      <Router>
+        <Fragment>
```

Comment:
we can remove this Fragment.

Fragment is like the ng-container in Angular, it groups several elements without rendering anything extra (like a div) in the dom

Reply:
interesting... 
http://cdn-webimages.wimages.net/051a555dee85db8539e36f1bace45586ae5f-wm.jpg?v=3